src/status_im/ui/screens/network_settings/events.cljs

```diff
@@ -0,0 +1,34 @@
+(ns status-im.ui.screens.network-settings.events
+  (:require [re-frame.core :as re-frame]
+            [taoensso.timbre :as log]
+            [status-im.ui.screens.navigation :as navigation]
+            [status-im.fleet.core :as fleet-core]
+            [status-im.utils.handlers :as handlers]
+            [status-im.utils.fx :as fx]))
+
+(fx/defn open-network-details
+  [cofx network show-warning?]
+  (fx/merge cofx
+            (when show-warning?
+              {:utils/show-popup {:title   "LES support is experimental!"
+                                  :content "Use at your own risk!"}})
+            (navigation/navigate-to-cofx :network-details {:networks/selected-network network})))
+
+(defn- not-supported-warning [fleet]
+  (str (name fleet) " does not support LES!\n"
+       "Please, select one of the supported fleets:"
+       (map name fleet-core/fleets-with-les)))
+
+(handlers/register-handler-fx
+ :ui/network-entry-clicked
+ (fn [cofx [_ network]]
+   (let [db                  (:db cofx)
```

Comment:
please move this to a fx/defn function and declare the event in `status-im.events` ns

Reply:
fixed